runtime/src/main/java/org/corfudb/runtime/view/AbstractReplicationView.java

```diff
@@ -1,129 +0,0 @@
-package org.corfudb.runtime.view;
```

Comment:
But we do, this code was marked deprecated, and the flexibility is retained in protocols